vuu-ui/packages/vuu-shell/src/user-settings/SettingsForm.tsx

```diff
@@ -164,23 +165,33 @@ export const SettingsForm = ({
     }
   };
 
-  // Change Handler for toggle and input buttons
+  // Change Handler for toggle and switch buttons
   const changeHandler = useCallback<FormEventHandler>(
     (event) => {
       const fieldName = getFieldNameFromEventTarget(event);
       const { checked, value } = event.target as HTMLInputElement;
-      onApplicationSettingChanged(fieldName, checked ?? value);
+      onSettingChanged(fieldName, checked ?? value);
     },
-    [onApplicationSettingChanged]
+    [onSettingChanged]
   );
 
   // Change handler for selection form controls
   const selectHandler = useCallback(
     (event: SyntheticEvent, [selected]: string[]) => {
       const fieldName = getFieldNameFromEventTarget(event);
-      onApplicationSettingChanged(fieldName, selected);
+      onSettingChanged(fieldName, selected);
     },
-    [onApplicationSettingChanged]
+    [onSettingChanged]
+  );
+
+  // Change Handler for input boxes
+  const inputHandler = useCallback<FormEventHandler>(
+    (event) => {
+      const fieldName = getFieldNameFromEventTarget(event);
+      const { value } = event.target as HTMLInputElement;
```

Comment:
this will always be a string. WIll need to convert to a numeric, if that is what the property represents